modules/auxiliary/scanner/http/xst.rb

```diff
@@ -0,0 +1,64 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Auxiliary
+
+  # Exploit mixins should be called first
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::WmapScanServer
+  # Scanner mixin should be near last
+  include Msf::Auxiliary::Scanner
+
+  def initialize
+    super(
+      'Name'        => 'Cross-Site Tracing (XST) Checker',
+      'Description' => 'Checks if the host is vulnerable to Cross-Site Tracing (XST)',
+      'Author'       => [ 'Jay Turla <@shipcod3>' ],
+      'License'     => MSF_LICENSE
+    )
+  end
+
+  def run_host(target_host)
+
+    begin
+      res = send_request_raw({
+        'version'      => '1.0',
+        'uri'          => '/<script>alert("TRACE Payload");</script>', #XST Payload
+        'method'       => 'TRACE',
+        'headers' =>
+        {
+          'Cookie' => "TRACE ME PLS",
+        },
+      }, 10)
+
+      if res.nil?
+        print_error("no repsonse for #{target_host}")
+      elsif (res.code == 200 and res.body =~/>alert/)
+        print_good("#{target_host}:#{rport}-->#{res.code}")
+        print_good("Response Headers:\n #{res.headers}")
+        print_good("Response Body:\n #{res.body}")
+        print_good("#{target_host}:#{rport} is vulnerable to Cross-Site Tracing\n")
+        report_note(
```

Comment:
You have used :data. Use :info instead. Take a look at https://github.com/wvu-r7/metasploit-framework/blob/f66c854ad694411794fbc15fd641c649d2e5abe7/modules/auxiliary/scanner/http/apache_mod_cgi_bash_env.rb for an example of report_vuln. The method itself is defined at https://github.com/rapid7/metasploit-framework/blob/10a6945737d5913640b38fde5ff46d3e015e1e98/lib/msf/core/db_manager/vuln.rb#L85